package.json

```
@@ -6,6 +6,9 @@
     "prettierTarget": "**/*.{js,ts,md,mdx,yml}"
   },
   "scripts": {
+    "test:e2e": "playwright test",
+    "test:debug": "playwright test --debug",
+    "test:unit": "jest tests/jest --maxConcurrency=8 --bail 1",
     "format": "prettier --write ${npm_package_config_prettierTarget}",
```

These commands exist in `django-ory-cloud/package.json` now and can be removed from here.

```suggestion
  "scripts": {
    "format": "prettier --write ${npm_package_config_prettierTarget}",
```